### PR TITLE
feat: create  component  details and about jobs

### PR DESCRIPTION
**Pull request by alishanti98**

### CODE REVIEW

File: client/src/components/jobDetails/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import './index.css';
+
+function jobDetails() {
```
Comment (LinaYahya) [inline client/src/components/jobDetails/index.tsx:5]: react component should named as `PascalCase`, So rename component to `JobDetails`

File: client/src/components/jobDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  return (
+    <div className="wrapper">
+      <div className="details">
+        <div className="top">
+          <h2>Job Details</h2>
+        </div>
+        <div className="content">
+          <p>
```
Comment (LinaYahya) [inline client/src/components/jobDetails/index.tsx:13]: there's more info related to job such as budget, duration, and category
Also, try to add them as props

File: client/src/components/jobDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import './index.css';
```
Comment (LinaYahya) [inline client/src/components/jobDetails/index.tsx:3]:
```suggestion
import './style.css';
```

File: client/src/components/jobDetails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        </div>
+        <div className="content">
+          <p>
+            Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. Necessitatibus quam, quasi
+            consequuntur ut perferendis tempora mollitia
+            assumenda nostrum accusamus maiores facere delectus
+            nobis nisi magni? Consequatur corporis saepe dolorum modi.
+          </p>
+          <Link to="/" className="link-btn">
```
Comment (LinaYahya) [inline client/src/components/jobDetails/index.tsx:20]: no need to have link here

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by mohammedagha27 — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**